virttest/qemu_devices/qcontainer.py

```diff
@@ -2252,6 +2252,8 @@ def chardev_define_by_params(self, chardev_id, params, file_name=None):
                  'port': params['chardev_port'],
                  'ipv4': params.get('chardev_ipv4'),
                  'ipv6': params.get('chardev_ipv6')})
+        if backend == 'tcp_socket':
+            chardev_param.update({'to': params.get('chardev_to', '')})
```

Comment:
Suggest removing the default value `''`.

Reply:
@luckyh updated, please help review again, thanks!